src/renderers/shared/fiber/__tests__/ReactIncrementalErrorHandling-test.js

```diff
@@ -935,4 +937,96 @@ describe('ReactIncrementalErrorHandling', () => {
     });
     expect(() => ReactNoop.flush()).toThrow('Error!');
   });
+
+  if (ReactDOMFeatureFlags.useFiber) {
```

Comment:
`ReactNoop` used in these tests is already Fiber-specific so this shouldn’t be necessary.